openff/interchange/components/interchange.py

```diff
@@ -367,6 +385,59 @@ def to_openmm_topology(
             ensure_unique_atom_names=ensure_unique_atom_names,
         )
 
+    def to_openmm_simulation(
+        self,
+        integrator: "openmm.Integrator",
+        combine_nonbonded_forces: bool = True,
+        add_constrained_forces: bool = False,
+        **kwargs,
+    ) -> "openmm.app.Simulation":
```

Comment:
Does it also make sense to have something like an `additional_forces` argument? It could take user-created Force objects and add them to the system before the simulation is created. Would be useful for forces that implement pressure coupling etc. Might be more discoverable than `simulation.system.addForce`?

Reply:
I'm neutral on this; I see the benefit to the user experience in making the `Simulation` object in one go, but in principle it opens the door to potential post-Interchange customization sneaking into the Interchange step

Reply:
https://github.com/openforcefield/openff-interchange/issues/730